src/src/App.js

```diff
@@ -50,6 +50,10 @@ export default function App() {
           <Route path="topics" element={<TopicsPage />} />
           <Route path="capabilities" element={<CapabilitiesPage />} />
           <Route path="ecr" element={<ECRPage />} />
+          <Route
+            path="capabilities/criticality"
+            element={<CapabilityCriticalityPage />}
+          />
```

Comment:
how does the footer work, do we need to check for cloud engineering status here too?

Reply:
This particular change has nothing to do with our footer.
But no, I don't think the footer needs any change (other than being glued to the bottom of the screen, but that is minor and not this scope🤷‍♂️ )